content/_index.md

```diff
@@ -1,9 +1,28 @@
 ---
-title: My Site
+title: AI Covenant
 toc: false
 ---
 
-This is the landing page.
+The use of "Artificial Intelligence" models affects the dynamics between people. We encourage mindful and considerate use of "AI" in our community that is grounded in our community values.
+
+Specific behaviors we do not allow:
+
+- Passing generated content off as your own
+- Arriving to a meeting with an AI assistant without asking the other participants
+- Sending your AI assistant to meetings in your place
+- [promoting the use of AI for everything (AI-ifying everything)]
+- - If the use of generative tools has been allowed, not verifying or validating the generated content.
+- [fully AI based production cycles]
+
+Specific behaviors we encourage/allow:
+
+- Generated content is specifically marked with the model + prompt (for example images)
+- Requesting consent of participants and organizers to use AI based tools.
+  - ...use "AI"
+  - ...share [information] with "AI"
```

Comment:
Inform other participants if data or content they have produced is going to be shared with an "AI" tool (e.g., for content summarization)